installer/cli/scenario1cli.py

```diff
@@ -0,0 +1,144 @@
+import click
+from installer.cli.click_required import Required
+from installer.configurators.jenkins import configure_jenkins
+from installer.configurators.bitbucket import configure_bitbucket
+from installer.configurators.sonarqube import configure_sonarqube
+from installer.helpers.terraform import exec_terraform_apply
+
+
+@click.command()
+# TODO Why do we not just accept the full url and strip the port and protocol if we're that picky?
+# TODO Why do we ask for all this Jenkins junk for scenario1? We really should just remove Jenkins from scenario 1.
+# If you are gonna BYOS you should install plugins and configure it yourself
+@click.option(
+    '--jenkins_userpass',
```

Comment:
--jenkins_userpass xxxxx xxxxxx is working fine. But if it not used as a parameter, what should be the user input for jenkins username and password? I had given 

> Jenkins userpass [()]: xxxxx xxxxxx

But its failing.

Reply:
With what error?


Might be a shell thing, might need to do `username\ pass` to escape the space character in the shell.

Reply:
If I enter it like, 
Jenkins userpass [()]: username password
 print(jenkins_userpass) would be like this, 
('u', 's', 'e', 'r', 'n', 'a', 'm', 'e', ' ', 'p', 'a', 's', 's', 'w', 'o', 'r', 'd')

Reply:
Added a workaround for this, it's a Click bug: https://github.com/pallets/click/issues/532

Reply:
Ok, But the parameter way failed. --jenkins_userpass username password

After, 
print(userpw)

the result is,

[usernamepassword]